app/controllers/rails/view_components_controller.rb

```diff
@@ -4,7 +4,6 @@
 
 class Rails::ViewComponentsController < Rails::ApplicationController # :nodoc:
   prepend_view_path File.expand_path("../../../lib/railties/lib/rails/templates/rails", __dir__)
-  prepend_view_path "#{Rails.root}/app/views/" if defined?(Rails.root)
```

Comment:
Do we need a CHANGELOG for this?

Reply:
I'll add an entry

Reply:
Will this work https://github.com/github/view_component/pull/296/commits/c1a2a949f521612611bad1aebf06da32908756db ?